CONTRIBUTING.md

```diff
@@ -30,44 +30,44 @@ Since the new GitHub Issue forms we only suggest you to include most information
 
 Unsure where to begin contributing to this project? You can start by looking through these beginner-friendly issues:
 
-- [Beginner issues](https://github.com/web3community/devprotocol.xyz/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) - issues which should only require a few lines of code, and a test or two.
-- [Help wanted issues](https://github.com/web3community/devprotocol.xyz/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) - issues which should be a bit more involved than `beginner` issues.
+*   [Beginner issues](https://github.com/web3community/devprotocol.xyz/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) - issues which should only require a few lines of code, and a test or two.
+*   [Help wanted issues](https://github.com/web3community/devprotocol.xyz/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) - issues which should be a bit more involved than `beginner` issues.
 
 ### 📣 Pull Requests
 
 The process described here has several goals:
 
-- Maintain the project's quality
-- Fix problems that are important to users
-- Engage the community in working toward the best possible!
-- Enable a sustainable system for maintainers to review contributions
+*   Maintain the project's quality
+*   Fix problems that are important to users
+*   Engage the community in working toward the best possible!
+*   Enable a sustainable system for maintainers to review contributions
 
 Please follow all instructions in [the template](https://github.com/web3community/devprotocol.xyz/blob/main/.github/pull_request_template.md)
 
 ## Style Guide for Git Commit Messages :memo:
 
 **How you can add more value to your contribution logs:**
 
-- Use the present tense. (Example: "Add feature" instead of "Added feature")
-- Use the imperative mood. (Example: "Move item to...", instead of "Moves item to...")
-- Limit the first line (also called the Subject Line) to *50 characters or less*.
-- Capitalize the Subject Line.
-- Separate subject from body with a blank line.
-- Do not end the subject line with a period.
-- Wrap the body at *72 characters*.
-- Use the body to explain the *what*, *why*, *vs*, and *how*.
-- Reference [issues](https://github.com/web3community/devprotocol.xyz/issues) and [pull requests](https://github.com/web3community/devprotocol.xyz/pulls) liberally after the first line.
-- Follow the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) guidelines
+*   Use the present tense. (Example: "Add feature" instead of "Added feature")
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 3](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8405536)
```suggestion
* Use the present tense. (Example: "Add feature" instead of "Added feature")
```